src/segment/segmentBase.js

```diff
@@ -55,3 +56,58 @@ export const segmentsFromBase = (attributes) => {
 
   return [segment];
 };
+
+export const addSegmentsToPlaylist = (playlist, sidx, baseUrl) => {
```

Comment:
what's the different between `playlist.sidx` and `sidx` here?

Reply:
`playlist.sidx` is the sidx information (where to get the box, the range, e.c.t) and `sidx` is the parsed box

Reply:
Might be a good comment to add to this function

Reply:
added